com.unity.multiplayer.mlapi/Editor/NetworkObjectEditor.cs

```diff
@@ -39,7 +39,7 @@ public override void OnInspectorGUI()
 
                 EditorGUILayout.EndHorizontal();
             }
-            else if (m_NetworkObject.IsSpawned)
+            else if (EditorApplication.isPlaying && m_NetworkObject.IsSpawned)
```

Comment:
Same: `&& m_NetworkObject` in the middle ?

Reply:
^ see above